prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1707,9 +1707,21 @@ static inline void rdm_ep_poll_ibv_cq(struct rxr_ep *ep,
 		case IBV_WC_RECV:
 			connid = rxr_pkt_connid_ptr(pkt_entry);
 			if (!connid) {
-				FI_WARN(&rxr_prov, FI_LOG_EP_CTRL, "No connid in packet header");
-				rxr_pkt_handle_recv_error(ep, pkt_entry, FI_EINVAL, FI_EINVAL);
-				break;
+				FI_WARN_ONCE(&rxr_prov, FI_LOG_EP_CTRL,
+					     "An incoming packet does have connection ID in its header.\n"
+					     "This means the peer is using an older version of libfabric.\n"
+					     "The communication can continue but it is encouraged to use\n"
+					     "a newer version of libfabric. See section 4.4 of\n"
+					     "https://github.com/ofiwg/libfabric/blob/main/prov/efa/docs/efa_rdm_protocol_v4.md\n"
```

Comment:
I'd drop the URL to our protocol doc as the average user probably won't want to read that.

Reply:
Thank you! dropped in new revision